ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -30,14 +30,26 @@
                 active-class="selected-radio"
                 v-model="HomeOwnerPartyTypes.OWNER_IND"
               />
-              <v-radio
-                id="executor-option"
-                class="executor-radio px-4"
-                label="Executor"
-                active-class="selected-radio"
-                :disabled="isTransferDueToDeath"
-                v-model="HomeOwnerPartyTypes.EXECUTOR"
-              />
+              <v-tooltip
+                v-if="isTransferDueToDeath"
```

Comment:
Minor thing, perhaps worth checking with design. 
I believe they wanted to show these radio options for Sale or gift but disabled. 
Could be wrong but this looks like it just hiding it vs disabling it.

Reply:
I'll check the designs

Reply:
Looks like disabled vs hidden for TRAND.

![Screen Shot 2023-03-27 at 1 33 40 PM](https://user-images.githubusercontent.com/53542131/228060157-bff5e62e-dfc2-4233-8fa9-fc0c47639d70.png)

Figma: 
https://www.figma.com/file/YcXMXw581zx7YxR34gkKvs/Wireframes---MHR-Transfers---Jan30-2023?node-id=161-31011&t=zz1PSvmUzTbMn99M-0

Reply:
Yep just found that. Will change